Implement 'create', 'update', 'createOrUpdate', and 'delete' methods (#1)
SynergiTech/laravel-salesforce: README.md

## Exceptions

By default [omniphx/forrest](https://github.com/omniphx/forrest) typically throws a single exception with more detail contained within a JSON encoded string.

> We've wrapped a fair few with our own exceptions to help with debugging.

Reviewer: ```suggestion
We've wrapped some with our own exceptions to help with debugging.
```
